### PR TITLE
Make Container Ports Optional:

### PR DESCRIPTION
**Pull request by EmmanuelOgiji**
- feat: Add if statement to remove container ports if .Values.port = null

### CODE REVIEW

File: charts/standard-application-stack/templates/deployment.yaml [hunk 1]
```diff
@@ -209,12 +209,15 @@ spec:
             {{- with .Values.envFrom }}
             {{- toYaml . | nindent 12 }}
             {{- end }}
+          {{- if eq .Values.port null}}
```
Comment (nabadger) [inline charts/standard-application-stack/templates/deployment.yaml:212]: you can use `ne` and then include the whole port section (and ditch the `else`)

Comment (nabadger) [reply charts/standard-application-stack/templates/deployment.yaml:212]: or even better 
```
{{-if .Values.port }} 
...
```

**Comment by barrymars** [comment]
general note on helm chart development

- Update CHANGELOG.md
- Update Chart.yaml with the new chart version
- Run `helm-docs` to regenerate docs

File: charts/standard-application-stack/Chart.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.4
+version: 2.1.5
```
Comment (barrymars) [inline charts/standard-application-stack/Chart.yaml]: doesn't match the version you put in the changelog

**Review by barrymars — APPROVED** [review]

**Pull request merged**